<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate Network Attributes</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>18</y>
     <width>709</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Input shapefile with subnetworks:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineInput">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Shapefile representing a stream network in which subnetworks have already been identified by the &lt;span style=&quot; font-weight:600;&quot;&gt;Find Subnetworks&lt;/span&gt; tool.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnBrowseInput">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="chkNetworkErrors">
   <property name="geometry">
    <rect>
     <x>531</x>
     <y>63</y>
     <width>196</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Network topology errors include duplicates, upstream flow direction, and crossed reaches.</string>
   </property>
   <property name="text">
    <string>Find network topology errors?</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>96</y>
     <width>676</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="lblOutput">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="text">
       <string>Processing output directory:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="lineOutput">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;A folder or directory which where tool outputs will be written. The outputs include:&lt;/p&gt;&lt;p&gt;- stream network line shapefile, with network IDs added as a new attribute field&lt;/p&gt;&lt;p&gt;- stream network nodes point shapefile, representing nodes within the network&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnBrowseOutput">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>333</x>
     <y>144</y>
     <width>395</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="btnHelp">
      <property name="text">
       <string>Help</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnReset">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnRun">
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="text">
       <string>Run</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnClose">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>192</y>
     <width>712</width>
     <height>307</height>
    </rect>
   </property>
   <property name="title">
    <string>Processing results</string>
   </property>
   <widget class="QPlainTextEdit" name="txtResults">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>24</y>
      <width>679</width>
      <height>229</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExportToTxt">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>264</y>
      <width>97</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Export to TXT</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
